src/utils/Resizer.js

```diff
@@ -133,6 +135,25 @@ define(function (require, exports, module) {
         return $(element).is(":visible");
     }
 
+    function _isPercentage(value) {
+        return !$.isNumeric(value) && value.indexOf('%') > -1;
+    }
+
+    function _percetageToPixels(value, total) {
```

Comment:
Your typo is `percetageToPixels` -> `percentageToPixels`. Also in all the function calls.